sklearn/model_selection/tests/test_validation.py

```diff
@@ -282,6 +282,16 @@ def test_cross_val_score():
     assert_raises(ValueError, cross_val_score, clf, X_3d, y2)
 
 
+def test_cross_val_score_sparsetarget():
+    X_, y_ = make_multilabel_classification(n_samples=100,
+                                            return_indicator=True,
+                                            random_state=0)
+    y_sparse = csr_matrix(y_)
+    clf = MockClassifier(a=0)
```

Comment:
I'd rather use a real classifier and assert `np.allclose(cross_val_score(clf, X_, y_sparse, cv=5), cross_val_score(clf, X_, y_sparse.toarray(), cv=5))`

Reply:
Then it should be tested with a classifier which accepts a sparse y (which one?). During the sprint, @GaelVaroquaux suggested to allow for sparse y in `RidgeClassifier` but that should be another PR, shouldn't it?